lib/drab.ex

```diff
@@ -213,6 +213,15 @@ defmodule Drab do
     {:noreply, state}
   end
 
+  # pass any other handle_info to the Commander `handle_info_message/2` callbacks,
+  # which expect `message` and `socket` params (no `state` reply).
+  def handle_info(message, state) do
+    state.socket
+    |> get_commander()
+    |> apply(:handle_info_message, [message, state.socket])
```

Comment:
This is synchronous!  Definitely shouldn't be!  Drab has async calls in this module, should follow the same pattern as them.

Reply:
Also, why is this name of `handle_info_message` hardcoded?  Although I like hardcoded names, that's not Drab's way.

Reply:
it is hardcoded as well as GenServer name `handle_info/2' callback is. I've choose not to call it just "handle_info" because though its role is similar, it is not identical (because it receive a `socket` instead of a `state`,  and doesn't return a `state`. Naming is difficult, so certainly we can choose another better name (actually I don't like too much 'handle_info_message' but after thinking a lot I couldn't find a better one). It have to be hardcoded because there is no other way, as far as I know, to have many of this kind of handlers (each one with it own pattern matching on message), in a Commander.